src/index/visitor.rs

```diff
@@ -452,6 +452,16 @@ fn visit_data_type(
                 information,
                 nature: TypeNature::Any,
             });
+            let global_init_name = format!("{}__init", name);
```

Comment:
I think we build this at 3 locations. Can we get a single util-method where we determine that name?

Reply:
Moved it to a helper method in index.rs